lang/src/org/partiql/lang/ast/passes/RewriterTransformBridge.kt

```diff
@@ -0,0 +1,28 @@
+package org.partiql.lang.ast.passes
```

Comment:
Same comment about duplicate code here. Am I missing something or is this due to the other PR not being committed and we will clean the 2 PRs accordingly before commit to master?

Reply:
Sorry for the confusion again. The other visitor transform PR (#298) should be merged first into the `visitor-transform` feature branch. Once all the visitor transform PIG migrations are done (i.e. committed into the `visitor-transform` feature branch), I believe there will just be one commit into master.